src/chart/radar/RadarView.ts

```diff
@@ -241,6 +241,8 @@ class RadarView extends ChartView {
                 else {
                     symbolPath.useStyle(itemStyle);
                     symbolPath.setColor(color);
+                    // fix display of empty symbol is not normal
+                    symbolPath.__isEmptyBrush && (symbolPath.style.lineWidth = 0.5);
```

Comment:
I don't think it's good to fix the width of the border. Especially to a very thin value that may have display issues on the 1x DPR monitor.